src/components/pages/Project/Project.js

```diff
@@ -1,20 +1,80 @@
-import React from "react";
-import "../../../../node_modules/bootstrap/dist/css/bootstrap.min.css";
+import React, { Component } from "react";
+import Navbar from "../../Nav/Nav";
+import { GoogleMap, LoadScript, Marker } from "@react-google-maps/api";
 import "./styles/Project.css";
 
-function Project(props) {
+const mapOptions = {
+  disableDefaultUI: true,
+  zoomControl: true,
+};
+
+const markers = [
+  { lat: 52.479738, lng: -1.903979 },
+  { lat: 52.474876, lng: -1.904408 },
+  { lat: 52.474744, lng: -1.888884 },
+];
+const engineers = ["Joe", "Niro", "Ian", "Meedaxa"];
+const center = {
+  lat: 52.479161,
+  lng: -1.895446,
+};
+
+const Projects = () => {
   return (
-    <div className="d-block">
-        <h1>Project name</h1>
-        <h2>Risks:</h2>
-        <li><a>Risk 1</a></li>
-        <li><a>Risk 2</a></li>
-        <li><a>Risk 3</a></li>
-        <div>
-          <h3>Map placeholder</h3>
+    <>
+      <Navbar />
+      <br />
+      <div className="map">
+        <div className="map-c">
+          <h1>Big Ben</h1>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            Accusantium id cumque est dolores voluptatibus.
+          </p>
+          <div className="det">
+            <p>
+              <i className="fa fa-map-marker"></i> <h2>Location</h2>
+              Westminster, London SW1A 0AA
+            </p>
+            <p>
+              <i class="fas fa-user-tie"></i>
+              <h2>Client</h2>
+              Arup
+            </p>
+            <div className="row">
+              <div className="col-sm-6">
+                <p>
+                  <i className="far fa-clock"></i>
+                  <h2>Start Date</h2><span style={{ marginLeft: 35 }}>6/12/19</span>
+                </p>
+              </div>
+              <div className="col-sm-6">
+                <p>
+                  <i className="far fa-clock"></i>
+                  <h2>End Date</h2>
+                  <span style={{ marginLeft: 35 }}>12/11/20</span>
+                </p>
+              </div>
+            </div>
+            <p>
+              <i className="fas fa-users"></i>
+              <h2>Team Members</h2>
+              {engineers.map((users) => (
+                <li className="users">{users}</li>
+              ))}
+            </p>
+          </div>
         </div>
-    </div>
+        <LoadScript googleMapsApiKey="AIzaSyDJyhMJUVajW8unTiFseBQPMlxmW1NltHM">
```

Comment:
This needs to be stored in environment variables

Reply:
resolved